### ISSUE TITLE
place a common thread through all the lessons

### ISSUE THREAD

**Issue by yarikoptic** [ReproNim/module-reproducible-basics#6]
@ReproNim/trd-training 
Examples and exercises could reflect back to the simple workflow paper thus providing a real life example of applicability of the presented/learned skills, and bind together all modules (not only the reproducible-basics)

**Comment by jdkent** [comment]
related to this, in the [git-novice](http://swcarpentry.github.io/git-novice/03-create/index.html) lesson, there is a story about wolfman and dracula sending a planetary lander to mars. I have one slightly thought out idea for a cast of characters for the repronim lessons, and another option I haven't put thought into, but might be interesting.
(I apologize in advance for all the horrible jokes)

## Option 1: The Amenders (Parody of Avengers)

**Dr. Green** (OS-Windows): An intelligent professor that received their PhD an unspecified number of years ago. They are normally an excellent communicator, but when working on things outside a GUI, they typically get frustrated and short tempered. 

**Blue Window** (OS-Windows): Research Scientist that primarily works in windows. Knowledgeable of powershell, and can spin up a Virtual Machine if necessary.

**Metal Person** (OS-Linux): Bit of a hardware junkie, often comes up with technology solutions that are not accessible to other lab members.

**Spyder Person** (OS-MAC):  Matlab convert, recognizes the value of python and is actively learning how to use python effectively while trying to maintain the "matlab" feel.

**Captain Country** (OS-MAC): Religiously on twitter, Captain Country their finger on the heartbeat of what's going on in open science by following 1000s of great scientists over the globe. However, sometimes Captain Country has difficulty curating the information to suggest what technologies would be most applicable/impactful to solve the practical problems in lab.

### Vague Villain/Obstacle:
**TanToes**: Working in a lab by a sunny coastline, the lab members often find themselves on the beach with no memory of what they were working on. Will they have documented their work well enough to retrace their steps and build on top of what they have already done?

Their Mission: Create a Simple Reproducible Workflow.

The characters/learner profiles can be fleshed out/changed to fit the target audience better. I haven't include sex/gender information yet.


## Option 2: Parody of [Portal](https://en.wikipedia.org/wiki/Portal_(video_game)) 

The main character has to solve a bunch of reproducbility problems.

Other thoughts/ideas/comments/criticisms are most definitely welcome.

**Comment by yarikoptic** [comment]
hm, that  is quite a "different" common thread idea... although having roles etc might be beneficial I am not sure if it wouldn't be too distracting.  Let's see what others think

**Comment by jdkent** [comment]
I agree, there is a thin line to straddle to make the characters useful without being distracting in the lessons/episodes.

The characters may have the greatest impact as an entertaining way to generate [learner profiles](https://cdh.carpentries.org/deciding-what-to-teach.html#learner-profiles) that would only be used internally to create the lessons and they would not make their way into the lessons themselves.

**Comment by ninapreuss** [comment]
I defer to the trainers!

-- 

Nina Preuss, MBA, PMP, Contractor
TCG, Inc. - Positively Distinct - CMMI-DEV Level 3 - CMMI-SVC Level 3 - ISO
9001:2015
Tel: 202-742-8479 | Mobile: 410-991-9337

**Comment by jbpoline** [comment]
I like the idea of having some learner profiles !
cheers
JB
